es6-classes/circle.js

```diff
@@ -4,8 +4,8 @@ class Circle extends Shape {
   constructor(radius) {
     super(radius);
     this.radius = radius;
-    this.area = Math.PI * radius ** 2;
-    this.perimeter = 2 * Math.PI * radius;
+    super.area = Math.PI * radius ** 2;
+    super.perimeter = 2 * Math.PI * radius;
```

Comment:
Not quite! You're not setting the `area`  & `perimeter` properties directly on `super`. You're supposed to call `super` and pass in the data that you want `area` & `perimeter` to be. When you do `super()` you are calling your `Shape` constructor function, which will generate the `area` & `perimeter` properties for you. So you just need to pass data into that function, to set it's values